ben_node:
  ros__parameters:
    debug_trigger: false
    verbose_trigger: false
    fsm_timer_callback_ms: 2000
    validation_average_window: 10
    validation_do_second_pass: false
    validation_timeout_sec: 20 # For no timeout, set to 0